src/components/SQForm/SQFormIconButton.tsx

```diff
@@ -1,17 +1,30 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-
 import {useFormButton} from './useFormButton';
 import {IconButton} from 'scplus-shared-components';
 
+interface SQFormIconButtonProps {
```

Comment:
Like we did for `onClick?`
We need to add a `?` at the end of a property if it is not a required `prop`